scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -304,7 +304,7 @@ def run_build(self, args=[]):
                     log.write(log_msg)
 
             if log_msg:
-                overflow_found = re.findall("region `(FLASH|ROM|RAM|ICCM|DCCM|SRAM|dram\\d_\\d_seg)' overflowed by", log_msg)
+                overflow_found = re.findall("region [`'](FLASH|ROM|RAM|ICCM|DCCM|SRAM|dram\\d_\\d_seg)':? overflowed by", log_msg)
```

Comment:
In your example, the line is broken into two:
```
ld.lld: error: section 'noinit' will not fit in region 'RAM': overflowed \
by 6624 bytes
```
 Shouldn't there be some line-joining logic? It seems like the regex would fail here, so I may be missing something.

Reply:
Apologies for the confusion here, this is due to me manually messing with the output. I was afraid of CI complaining about line length in the commit message so I manually wrapped the error messages here--the actual error messages don't have the `\` and line break.

I modified this in the description above and added a note explaining this to hopefully make this more clear!